### PR TITLE
added basic readme instructions

### PR DESCRIPTION
**Pull request by dandrewgarvin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
...
+## Setup Instructions
+
+This template is intended to be an out-of-the-box way to set up a new project in a matter of minutes. Once a repo has been created off of this template, the user can clone the new repo to their preferred machine, run `npm install`, change the needed variables, and then run `npm start` and `npm run start:server` to get the project running!
+
+### Create new repo
+
+Do not use this template directly. You must create a new repo on your own github account using this template (this is all done on the github.com website).
+
+Once you have created a new repo off of this template, clone and download it to your computer.
+
```
Comment (brennongs) [inline README.md]: maybe link to the directions for how to do that?

**Review by brennongs — CHANGES_REQUESTED** [review]
Seems pretty good for what is here. /shrug

File: README.md (hunk 1 above)
Comment (dandrewgarvin) [reply README.md]: added in PR update

**Review by brennongs — APPROVED** [review]
looks good

**Pull request merged**